spec/controllers/auth/registrations_controller_spec.rb

```diff
@@ -232,6 +232,8 @@
     it 'does nothing if user already exists' do
       Fabricate(:account, username: 'test')
       subject
+
+      pending 'Implement noop check'
```

Comment:
This one you could wrap the `subject` call in a block and check that `User.count` does not change before vs after the POST to create?